lib/cli/ui/spinner/spin_group.rb

```diff
@@ -97,6 +101,7 @@ def render(index, force = true, width: CLI::UI::Terminal.width)
           # * +title+ - title to change the spinner to
           #
           def update_title(new_title)
+            @always_full_render = new_title =~ Formatter::SCAN_WIDGET
```

Comment:
Can you document somewhere the intention behind `always_full_render` vs `force_full_render`?